cogito/ghcommitsink_test.go

```diff
@@ -47,17 +51,21 @@ func TestSinkGitHubCommitStatusSendFailure(t *testing.T) {
 		http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 			w.WriteHeader(http.StatusTeapot)
 		}))
+	gitHubSpyURL, err := url.Parse(ts.URL)
+	if err != nil {
+		t.Fatalf("error parsing SpyHttpServer URL: %s", err)
```

Comment:
same

Reply:
Fixed in: https://github.com/Pix4D/cogito/commit/66f1cf52187b4780f4fb0775009bab43f2a2155d